Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -9246,6 +9244,79 @@ def get_template_stacks(
 
 
+def get_global_counters(topology: Topology, device_filter_string: Optional[str] = None) -> ShowCounterGlobalCommmandResult:
+    """
+    Gets global counter information from all the PAN-OS firewalls in the topology
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    return FirewallCommand.get_counter_global(topology, device_filter_string)
+
+
+def get_bgp_peers(topology: Topology, device_filter_string: Optional[str] = None) -> ShowRoutingProtocolBGPCommandResult:
+    """
+    Retrieves all BGP peer information from the PAN-OS firewalls in the topology.
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    return FirewallCommand.get_bgp_peers(topology, device_filter_string)
+
+
+def get_available_software(topology: Topology, device_filter_string: Optional[str] = None) -> SoftwareVersionCommandResult:
+    """
+    Check the devices for software that is available to be installed.
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    return UniversalCommand.get_available_software(topology, device_filter_string)
+
+
+def get_ha_state(topology: Topology, device_filter_string: Optional[str] = None) -> List[ShowHAState]:
+    """
+    Get the HA state and associated details from the given device and any other details.
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    """
+    return FirewallCommand.get_ha_status(topology, device_filter_string)
+
+
+def get_jobs(topology: Topology, device_filter_string: Optional[str] = None, status: Optional[str] = None,
+             job_type: Optional[str] = None, id: Optional[str] = None) -> List[ShowJobsAllResultData]:
+    """
+    Get all the jobs from the devices in the environment, or a single job when ID is specified.
+
+    Jobs are sorted by the most recent queued and are returned in a way that's consumable by Generic Polling.
+    :param topology: `Topology` instance !no-auto-argument
+    :param device_filter_string: String to filter to only show specific hostnames or serial numbers.
+    :param status: Filter returned jobs by status
+    :param job_type: Filter returned jobs by type
+    :param id: Filter by ID
+    """
+    _id = arg_to_number(id)
+
+    return UniversalCommand.show_jobs(
+        topology,
+        device_filter_string,
+        job_type=job_type,
+        status=status,
+        id=_id
+    )
+
+
+def download_software(topology: Topology, version: str,
+                      device_filter_string: Optional[str] = None, sync: Optional[bool] = None) -> DownloadSoftwareCommandResult:
```

Comment:
i think i mislead you, since ```sync``` has a default value of ```False``` that means it should be this way:

```suggestion
                      device_filter_string: Optional[str] = None, sync: bool = False) -> DownloadSoftwareCommandResult:
```
The sync can never be ```None``` because it has a default value.